src/clj/rems/css/styles.clj

```diff
@@ -202,6 +204,65 @@
                                   :margin-top 0
                                   :margin-bottom (u/px 16)})
 
+(defn- remove-nil-vals
+  "Recursively removes all keys with nil values from a map."
+  [obj]
+  (cond
+    (record? obj) obj
+    (map? obj) (let [m obj
+                     m (->> m
+                            (map-vals remove-nil-vals)
+                            (remove-vals nil?))]
+                 (when (seq m)
+                   m))
+    (sequential? obj) (let [coll obj]
+                        (map remove-nil-vals coll))
+    :default obj))
+
+(deftest test-remove-nil-vals
+  (testing "empty"
+    (is (= nil
+           (remove-nil-vals {}))))
+  (testing "flat"
+    (is (= nil
+           (remove-nil-vals {:a nil})))
+    (is (= {:a 1}
+           (remove-nil-vals {:a 1})))
+    (is (= {:a false}
+           (remove-nil-vals {:a false})))
+    (is (= {:a ""}
```

Comment:
Maybe empty strings should be considered as nils because they make no sense in the resulting CSS?

Reply:
Maybe. We shouldn't have any empty strings right now, so it's safer to not touch them. If some day empty strings appear somewhere, it might blow up and produce confusing bugs if they are removed from places where we don't want to remove them.

Reply:
Then make it blow up if it meets empty strings?